src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1546,6 +1551,9 @@ def create_vm(vm_name, resource_group_name, image=None,
     client = get_mgmt_service_client(ResourceManagementClient).deployments
     properties = DeploymentProperties(template=template, parameters={}, mode='incremental')
     if validate:
+        from pprint import pprint
+        import sys
+        pprint(template, sys.stderr)
```

Comment:
No, I actually put this in here just for you. 😄 You mentioned how pulling the template out of the --debug output was frustrating so I figured this would be more helpful.  Thoughts?